data/174/607/202/5/1746072025.geojson

```diff
@@ -0,0 +1,59 @@
+{
+  "id": 1746072025,
+  "type": "Feature",
+  "properties": {
+    "edtf:cessation":"uuuu",
+    "edtf:inception":"uuuu",
+    "geom:area":0.000342,
+    "geom:bbox":"46.653878,24.674924,46.678195,24.708434",
+    "geom:latitude":24.691177,
+    "geom:longitude":46.666751,
+    "iso:country":"SA",
+    "lbl:latitude":24.691723,
+    "lbl:longitude":46.666567,
+    "mps:latitude":24.691723,
+    "mps:longitude":46.666567,
+    "mz:hierarchy_label":1,
+    "mz:is_current":1,
+    "mz:is_funky":0,
+    "name:ara_x_preferred":[
+        "\u0627\u0644\u0645\u0639\u0630\u0631 \u0627\u0644\u0634\u0645\u0627\u0644\u064a"
+    ],
+    "name:eng_x_preferred":[
+        "Al Maathar Al Shemali"
+    ],
+    "name:eng_x_variant":[
+        "\u0627\u0644\u0645\u0639\u0630\u0631 \u0627\u0644\u0634\u0645\u0627\u0644\u064a"
```

Comment:
This name is Arabic, not an English variant

Reply:
Ah good catch. I thought I caught all of these cases in a previous commit, but it looks like a few remain. I'll add a new commit to update these.

Reply:
fixed in 27ea53d